<?xml version="1.0"?>

<launch>
  <node name="mask_rgb" pkg="cob_people_object_detection_tensorflow" type="mask_rgb.py" output="screen"  >
#launch-prefix="xterm -e python -m pdb "
  </node>
</launch>
